src/Scenes/CheckLocation.fs

```diff
@@ -20,7 +20,7 @@ type Status =
 | Changed
 | Error of string
 
-type Model =
+type [<Pojo>] Model =
```

Comment:
It is for React for the Web, as it only accepts plain-JS objects as view props (I'd have to check, maybe they changed that again). I'm not sure about React Native, you can try removing the attribute and see if it still works and/or whether you get any warning in the Debugger console.